docs/workforce-management-framework/topics/con-datasync-module.adoc

```diff
@@ -31,6 +31,6 @@ The *Datasync Client App* is based on the link:https://github.com/feedhenry/fh-s
 In order to work, the *Datasync Client App* requires the *Datasync Cloud App* to be running on an _Express_ web server.
 The _Data Synchonization Library_ can be used for offline storage of the _Client App's_ data.
 
-For instructions on how to implement the *Datasync Client App*, see the *Datasync Client App* link:{WFM-RC-CoreURL}{WFM-RC-Branch}/client/datasync-client/README.md[README.MD]
+For instructions on how to implement the *Datasync Client App*, see the *Datasync Client App* link:{WFM-RC-CoreURL}{WFM-RC-Branch}/client/datasync-client/README.md[README]
```

Comment:
@austincunningham - What you mean by *Datasync client app*?

Reply:
two seperate npm packages involved in Datasync , client and cloud. if App isn't the right description then Open to suggestions.